src/util.ts

```diff
@@ -0,0 +1,87 @@
+import { Logger } from './logger';
+
+const temp = require('temp');
+const fs = require('fs');
+const logger:Logger = Logger.getInstance();
+
+
+export class TempFileHandler {
+    track: any;
+
+    create(TextEditor) {
+        let tempFile = temp.openSync({suffix: '.py'});
```

Comment:
I have to think about that. In my case the path looks like below:
`/tmp/116614-30510-d9bv3h.py`
The name is created by "temp" module.